VeterinaryValeriya/Program.cs

```diff
@@ -105,5 +59,86 @@ static void PrintEnumOfAnimalTypes()
             Console.WriteLine("<3> for {0}", TypeOfAnimal.Fish.ToString());
             Console.WriteLine();
         }
+
+        public static char GetUserInput()
+        {
+            return Console.ReadLine()[0];
+        }
+
+        private static void ShowAboutCompany ()
+        {
+            Console.WriteLine("This is veterinary clinic for domestic animals:)");
+        }
+
+        private static void PutAnimalInClinic()
```

Comment:
I would split to 2 methods. First method will get information from user, second method will deal with logical data (inputTypeOfAnimal should be something like enum in this second method)